Src/AllocationCallbacks.cs

```diff
@@ -144,6 +144,7 @@ internal void ToNative(Native* native)
         /// <param name="allocationScope">
         /// Value specifying the allocation scope of the lifetime of the allocation.
         /// </param>
+        [UnmanagedFunctionPointer(CallingConvention.Winapi)]
```

Comment:
Why is this needed?

Reply:
The vulkan call is stdcall on windows and cdecl on non windows

Reply:
What's the default behavior?

Reply:
None, if you dont mark without UnmanagedFunctionPointer, WinApi behave as I described,  StdCall on windows Cdecl else where